tasks/variables.yml

```diff
@@ -0,0 +1,10 @@
+---
+# Variable configuration.
+- name: Include OS-specific variables (Debian).
+  ansible.builtin.include_vars: "{{ ansible_distribution }}-{{ ansible_distribution_version.split('.')[0] }}.yml"
+  when: ansible_os_family == 'Debian'
+
+- name: Define python_packages.
```

Comment:
This task winds up being triggered when not Debian (Ubuntu and RHEL) as well and failing.

I like the method you are using here however either all supported distros will need refactored to this style or you will need to add a conditional here for Debian as well.

I am also happy if you add a conditional here for now and want me to refactor the rest.

Reply:
Thank you very much for this!